nix/cabal-docspec.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+pkgs.stdenv.mkDerivation {
+  name = "cabal-docspec";
+
+  src = pkgs.fetchurl {
```

Comment:
Presumably you could use `fetchzip` here, instead. This would simplify your fetch a bit.

Reply:
apprently it doesn't like the xz compression:
```
[~] nix-prefetch-url --unpack https://github.com/phadej/cabal-extras/releases/download/cabal-docspec-0.0.0.20211114/cabal-docspec-0.0.0.20211114.xz
error: failed to open archive: Unrecognized archive format
```

(I am assuming `nix-prefetch-url --unpack` is the same as `fetchzip`)